### ISSUE TITLE
servlet-monitor should have apis for custom metrics

### ISSUE THREAD

**Issue by eabili0** [labbsr0x/servlet-monitor#20]
The lib should have apis for the creation and exposure of custom metrics, to make it easy for users to expose their custom system info.

**Comment by werberson** [comment]
The way to expose custom collectors on the same `/metrics` is registering the customs collectors on the monitor metrics registry `MonitorMetrics.INSTANCE.collectorRegistry`.

e.g.
```java
Counter.build().name("my_custom_metric")
     .help("custom metric registered and exposed by MonitorMetrics")
     .labelNames("type", "status", "custom_label")
     .register(MonitorMetrics.INSTANCE.collectorRegistry);
```

**Comment by eabili0** [comment]
I think this is too cumbersome. Couldn't we do it a little bit better?

**Comment by eabili0** [comment]
Antoher point is that doing that, we are not enforcing the Big Brother Protocol. We should always help the users to enforce that and stay compliant.

**Comment by eabili0** [comment]
Fixed by #22
